integration_tests/CMakeLists.txt

```
@@ -2617,3 +2617,4 @@ RUN(NAME lp_union_test_02 LABELS llvm llvm_wasm llvm_wasm_emcc) # No gfortran
 
 # LFortran extensions (unsigned int)
 RUN(NAME test_unsigned LABELS llvm llvm_wasm llvm_wasm_emcc) # No gfortran 
+RUN(NAME pointer_assignment_01 LABELS gfortran llvm)
```

```suggestion
RUN(NAME associate_xx LABELS gfortran llvm)
```

Let's have this name